libmount/src/lock.c

```diff
@@ -146,7 +146,7 @@ static int lock_simplelock(struct libmnt_lock *ml)
 	const char *lfile;
 	int rc;
 	struct stat sb;
-	const mode_t lock_mask = S_IRUSR|S_IWUSR|S_IRGRP|S_IROTH;
+	const mode_t lock_mask = S_IRUSR|S_IWUSR;
```

Comment:
Doesn't this break backwards compatibility for other users monitoring this file directly?

Reply:
We do not care ;-) It's a private library file, nowhere documented.